### ISSUE TITLE
Open Ended Question Type for LMZ Quiz

### ISSUE THREAD

**Issue by t4ggeek** [t4glabs/echolearn#3]
For each course (what we are calling an echo - you can see what an echo is here https://www.echolearn.org/support ) , we want to add just 2 open ended questions. no correct answer. just for them to write something.

**Comment by t4ggeek** [comment]
@wisharya Will this also move to frappe/school ?

**Comment by fossdot** [comment]
Yes, this can be tracked here https://github.com/frappe/school/issues/245.
